src/main/java/com/bynder/sdk/util/Utils.java

```diff
@@ -95,21 +96,32 @@ private static OkHttpOAuthConsumer createHttpOAuthConsumer(final String consumer
      * Creates an instance of {@link OkHttpClient}.
      *
      * @param consumer {@link OkHttpOAuthConsumer} instance.
+     * @param httpConnectionSettings settings for the http connection to Bynder
      *
      * @return {@link OkHttpClient} instance used for API requests.
      */
-    private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer) {
+    private static OkHttpClient createHttpClient(final OkHttpOAuthConsumer consumer, HttpConnectionSettings
+            httpConnectionSettings) {
         OkHttpClient.Builder httpClient = new OkHttpClient.Builder();
         httpClient.interceptors().clear();
         httpClient.addInterceptor(new SigningInterceptor(consumer));
 
         HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
         interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
         httpClient.addInterceptor(interceptor);
-
-        httpClient.readTimeout(30, TimeUnit.SECONDS);
-        httpClient.connectTimeout(30, TimeUnit.SECONDS);
-
```

Comment:
add it back

Reply:
why? :) it's there with the default http connectiion settings in HttpConnectionSettings.java

Reply:
you are right, I didn't notice :)